timeline-chart/src/time-graph-font-controller.ts

```diff
@@ -5,9 +5,14 @@ export class FontController {
     private fontNameMap: Map<number, Map<string, string>>;
     private fontColorMap: Map<number, string>;
     private defaultFontName: string = "LabelFont8White";
+<<<<<<< 0296bec1d7bad58aea8bdc94e79acd6b479926c3
```

Comment:
Those are merge conflict markers, please remove

Reply:
Done. Thank you.